### ISSUE TITLE
When will PHP 8.2 be supported？

### ISSUE THREAD

**Issue by AuroraYolo** [phpbrew/phpbrew#1347]
### Output

```
...
```

### Expected Result

```
...
```

### Command

Paste your phpbrew install command here (be sure to install with `--debug` flag, e.g. `phpbrew --debug install`)

    phpbrew --debug install php-5.x.x +openssl -- --....

### Build Log

    gist here

### Platform

OS:

    e.g. OS X 10.7, Ubuntu 14.04

Running PHP: 

Installing PHP:

**Comment by AuroraYolo** [comment]
?

**Comment by peter279k** [comment]
The issue is similar with #1345.

**Comment by AuroraYolo** [comment]
Will 8.2 be supported soon?

**Comment by peter279k** [comment]
Yes. Sorry. I'm busy at recent weeks. And I will inform you once the PHP 8.2 version is supported.

**Comment by AuroraYolo** [comment]
你使用微信吗？

**Comment by peter279k** [comment]
> 你使用微信吗？

No. I don't use that.
